main.py

```
@@ -1,44 +1,412 @@
+        if not broadcast_text:
+            bot.send_message(message.chat.id, "Usage: `/broadcast [your message]`", parse_mode='Markdown')
+            return
+        
+        sent = 0
+        failed = 0
+        
+        for user_id in user_data.keys():
+            try:
+                bot.send_message(user_id, f"📢 **HIT PRO Announcement**\n\n{broadcast_text}", parse_mode='Markdown')
+                sent += 1
+            except:
```

Ruff (0.15.7)

[error] Do not use bare `except`

(E722)